src/app/shared/components/text-tooltip/text-tooltip.component.ts

```diff
@@ -6,5 +6,5 @@ import { Component, Input } from '@angular/core'
   styleUrls: ['./text-tooltip.component.scss'],
 })
 export class TextTooltipComponent {
-  @Input() public text: string
+  @Input() public text?: string
```

Comment:
Why is this made optional?

Reply:
Because on many places we have conditional tooltip. That way we provide or not the tooltip and when we have a text we apply the styles to the div and show the tooltip if we don't have a text we don't show anything. That was discuss it with @LindvedKrvang in to the comments above.

```
<span *ngIf="text">
  {{ text }}
</span>

```